packages/web3-core/types/index.d.ts

```diff
@@ -456,3 +461,10 @@ export type provider =
     | AbstractProvider
     | string
     | null;
+
+export interface AccessTuple {
```

Comment:
The reason I did this way is because `web3-core` is `not` dependent on `web3-types`, but, the other way around (`web3-types` imports from `web3-core`). 
So if we move `Transaction` it won't be able to be used inside `web3-core`.
 Please let me know your thoughts :)

Reply:
@nikoulai and @Muhammad-Altabba this is 1.x PR and we dnt have `web3-types` package. are you guys referring 4.x here? PR lgtm , except I have question above.

Reply:
Sorry, my bad.